### ISSUE TITLE
interfacePCA

### ISSUE THREAD

**Issue by ellispatrick** [SydneyBioX/ClassifyR#41]
This broke when variables had zero variability. I need to add some filtering here in a sensible way.

**Comment by nick-robo** [comment]
@DarioS were you able to replicate this?

**Comment by DarioS** [comment]
No, it works fine for me now. Ellis should provide an example of the error or close this issue if it is no longer applicable.
